tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -113,6 +113,12 @@ generic_config_updater/test_eth_interface.py::test_replace_fec:
     conditions:
       - "platform in ['x86_64-kvm_x86_64-r0']"
 
+generic_config_updater/test_incremental_qos.py::test_incremental_qos_config_updates:
+  skip:
+    reason: "This test was not run on this hwsku type currently"
+    conditions:
+      - "not any(i in hwsku for i in ['2700', 'Arista-7170-64C', 'montara', 'newport'])"
```

Comment:
`hwsku in ['2700', 'Arista-7170-64C', 'montara', 'newport']` would be better?

Reply:
Yes, it would be better, but the logic from [test_incremental_qos.py](https://github.com/Azure/sonic-mgmt/blob/master/tests/generic_config_updater/test_incremental_qos.py#L204) requires checking part of hwsku name. So I've just moved this logic. Thanks!